package/MDAnalysis/core/dummy.py

```diff
@@ -19,9 +19,7 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
-"""Mock Universe and Topology for testing purposes
-
-Avoids import of MDAnalysis at base level because of Issue #344
+"""Mock Universe and Topology generated from scratch with default values
```

Comment:
I was thinking of saying that we could merge the behaviour of `make_Universe` into `mda.Universe` so you could do `mda.Universe(100, attrs=['charges', 'resnames'])`, but maybe `Universe.__init__` is already complicated enough....

`make_Universe` is a confusing function name, as it's also what `mda.Universe()` does.. Maybe call it `make_blank_Universe()` or something to designate it as a different type of universe creation